specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -0,0 +1,16019 @@
+        "operationId": "VirtualMachineScaleSets_ConvertToSinglePlacementGroup",
+        "description": "Converts SinglePlacementGroup property to false for a existing virtual machine scale set.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine scale set to create or update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VMScaleSetConvertToSinglePlacementGroupInput"
+            },
+            "description": "The input object for ConvertToSinglePlacementGroup API."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "x-ms-examples": {
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MaximumSet_Gen.json"
+          },
+          "VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen": {
+            "$ref": "./examples/compute/VirtualMachineScaleSets_ConvertToSinglePlacementGroup_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/setOrchestrationServiceState": {
+      "post": {
+        "tags": [
+          "VirtualMachineScaleSets"
+        ],
+        "operationId": "VirtualMachineScaleSets_SetOrchestrationServiceState",
+        "description": "Changes ServiceState property for a given service",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine scale set to create or update."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/OrchestrationServiceStateInput"
+            },
+            "description": "The input object for SetOrchestrationService
```

Comment:
this should be a string enum since it has constrained values according to the description

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/16535/files

Reply:
pinged feature owner for the requested change

Reply:
changed to enum